cranelift-reader/src/testfile.rs

```diff
@@ -55,3 +57,17 @@ pub struct Comment<'a> {
     /// Text of the comment, including the leading `;`.
     pub text: &'a str,
 }
+
+/// A cranelift feature in a test file preamble.
+///
+/// This represent the expectation of the test case. Before running any of the
+/// functions of the test file, the feature set should be compared with the
+/// feature set used to compile Cranelift. If there is any differences, then the
+/// test file should be skipped.
+#[derive(PartialEq, Eq, Debug)]
+pub enum Feature<'a> {
+    /// `feature "..."` lines
+    Has(&'a str),
+    /// `feature ! "..."` lines.
+    HasNo(&'a str),
```

Comment:
Light suggestion to rename these two "With" and "Without", maybe? Or alternatively, "Want" / "DontWant"?

Reply:
I renamed these to "With" and "Without". Thanks.